PWGJE/FlavourJetTasks/AliAnalysisTaskHFJetIPQA.h

```diff
@@ -223,7 +223,7 @@ class AliAnalysisTaskHFJetIPQA: public AliAnalysisTaskEmcalJet
     TH2D *fh2dCharmDeta; //!
     TH2D *fh2dBottomDeta; //!
 
-    THnSparse *fHLundIterative;//       iterative declustering
+    THnSparseF *fHLundIterative;//!       iterative declustering
```

Comment:
Also here the specification of the derived type in member declaration is not necessary, the problem was more the missing "!".